googletest/include/gtest/internal/gtest-param-util.h

```diff
@@ -396,6 +397,30 @@ typename ParamNameGenFunc<ParamType>::Type *GetParamNameGen() {
   return DefaultParamName;
 }
 
+// INTERNAL IMPLEMENTATION - DO NOT USE IN USER CODE.
```

Comment:
Don't need any of those at all, see suggestion in the googletest/include/gtest/gtest-param-test.h